server/src/test/java/io/crate/integrationtests/KillIntegrationTest.java

```diff
@@ -48,7 +48,7 @@
 import static org.hamcrest.Matchers.instanceOf;
 import static org.hamcrest.Matchers.is;
 
-public class KillIntegrationTest extends SQLTransportIntegrationTest {
+public class KillIntegrationTest extends BaseUsersIntegrationTest {
```

Comment:
I switched to `BaseUsersIntegrationTest` to get access to `executeAsNormalUser`. Alternative could be to either extend `SQLTransportIntegrationTest` with such a method or add one to `KillIntegrationTest`. 

Wasn't sure what's the optimal solution for this. Would you prefer one of the alternatives?

Reply:
I think it's fine this way :+1: